### PR TITLE
add dnsdad template

### PR DESCRIPTION
**Pull request by arya-dos**
We have a podcasting platform

In the podcasting platform where our customers create websites using a custom drag & drop builder tool feature on our platform. Our customer who created a website using our tool is facing difficulty in DNS mapping manually. So, we want to add functionality that deal with the DNS providers from and change the DNS.

I have 2 servers

dnsdad.com
podup.com (Podcasting platform)
What I’ll do and want to do,

I’ll create an API of dnsdad.com and whenever the customers wants to change the DNS, the podup will use the API and through the dnsdad.com the request will be sent to the DNS Providers.

### CODE REVIEW

**Comment by pawel-kow** [comment]
Still an issue with file name not matching service id.
```AssertionError: File name dnsdad.com.hosting.json not matching the pattern. Expected dnsdad.com.dnsdadrecords.json```

**Review by pawel-kow — CHANGES_REQUESTED** [review]
Rename file pls

**Review by arya-dos — COMMENTED** [review]
file name changed

File: dnsdad.com.dnsdadrecords.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "providerId": "dnsdad.com",
+    "providerName": "dnsdad.com",
+    "serviceId": "dnsdadrecords",
+    "serviceName": "Dnsdad Records",
+    "version": 1,
+    "logoUrl": "https://dnsdad.com/logo.svg",
+    "description": "Configure DNS records using DNSDAD",
+    "variableDescription": "IP address for the domain A, CNAME. Domain for TXT, MX",
```
Comment (pawel-kow) [inline dnsdad.com.dnsdadrecords.json:9]: Please review the variable description. It looks like not referring to the actual variables, like MX.

File: dnsdad.com.dnsdadrecords.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            "type": "CNAME",
+            "host": "%cnamehost%",
+            "pointsTo": "%domain%",
+            "ttl": 3600
+        },
+        {
+            "type": "TXT",
+            "host": "@",
+            "ttl": 3600,
+            "data": "%txt%"
```
Comment (pawel-kow) [inline dnsdad.com.dnsdadrecords.json:34]: What is this TXT record about?

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: dnsdad.com.dnsdadrecords.json (hunk 1 above)
Comment (arya-dos) [reply dnsdad.com.dnsdadrecords.json:9]: Actually, we planned for MX Record earlier and later removed the MX record but forget to remove MX record from Description. I update the Description.

**Review by pawel-kow — APPROVED** [review]

**Pull request merged**